runtime/cpp/emboss_defines.h

```diff
@@ -66,9 +66,17 @@
 //
 // By default, checks are only enabled on non-NDEBUG builds.  (Note that all
 // translation units MUST be built with the same value of NDEBUG!)
+//
+// The EMBOSS_SKIP_CHECKS parameter allows all CHECKs to be compiled out
+// without -DNDEBUG; this is useful for testing.
 #if !defined(EMBOSS_CHECK)
+#if EMBOSS_SKIP_CHECKS
```

Comment:
Should this be `#if defined(EMBOSS_SKIP_CHECKS)`? I'm a little rusty on preprocessor rules

Reply:
It probably should be for consistency with surrounding code.

Technically, they're almost identical -- `#if SYMBOL` is false if `SYMBOL` is undefined or evaluates to 0; `#ifdef SYMBOL` or `#if defined(SYMBOL)` is only false if `SYMBOL` is undefined.  Passing `-DSYMBOL` implicitly `#defines` `SYMBOL` to be `1`.

Reply:
Ah thanks for the refresher!